src/components/modal/Modal.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Modal.css';
+
+export const Modal = ({ handleClose, modalClass, children }) => {
+  const showHideClassName = modalClass
+    ? 'modal display-block'
+    : 'modal display-none';
+  return (
+    <div className={showHideClassName}>
+      <section className="modal-main">
+        <button className="button_close" onClick={handleClose}>
```

Comment:
the closing button for the 'added item' modal did not work for me 🤔

Reply:
Not sure why that can be I just tried it on the live site and it worked. Could you try again and tell us if you get an error of some type?  :)

Reply:
Hi [commit](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/commit/056255cede14fa4e7b9997fae89418a68ee57d29) fixed a little more style of  button close, but I proved at local and working fine

Reply:
yep, now it works fine. not sure why but when I clicked on it it would not close. Good job guys!